pkg/kubelet/apis/podresources/server_v1alpha1_test.go

```diff
@@ -62,6 +62,10 @@ func (m *mockProvider) GetAllocatableCPUs() []int64 {
 	return args.Get(0).([]int64)
 }
 
+func (m *mockProvider) IsExclusive(podUID, containerName string) bool {
+	args := m.Called(podUID, containerName)
+	return args.Get(0).(bool)
+}
```

Comment:
This looks like a leftover of a past revision

Reply:
Good catch! Thanks, fixed it.